### PR TITLE
Adding AVL tree

### PR DESCRIPTION
**Pull request by akhil-maker**
#1 
Hi !
I am adding AVL tree for data structure in tree.
Do give a review....

### CODE REVIEW

**Review by Sabine91 — CHANGES_REQUESTED** [review]
Please make sure what changes u have made before opening a pull request, seems like u have also changed something in the Fenwick tree, if that was intended then please add it in your description or title, and why have u added a prexisting line again in the main readme file.

#### What to do?
* Make sure u are only adding AVL tree and not messing up anything else.
* Remove any unecessary changes to the main readme.md

**Comment by akhil-maker** [comment]
@Sabine91 
I have only added AVL tree and not make any changes other than this.
You can check in the commit that there are not changes made in Fenwick tree.

File: readme.md [hunk 1]
```diff
@@ -200,6 +200,9 @@ The time complexity of an algorithm estimates how much time the algorithm will u
 ### [Space Complexity](Algorithms/Space%20Complexity/readme.md)
 Space complexity refers to the total amount of memory space used by an algorithm/program, including the space of input values for execution. Calculate the space occupied by variables in an algorithm/program to determine space complexity.
 
+### [Space Complexity](Algorithms/Space%20Complexity/readme.md)
```
Comment (shhossain) [inline readme.md:203]: Space complexity section in two places.

**Review by shhossain — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (akhil-maker) [reply readme.md]: Corrected it now
Please review it.
Now only AVL Tree is added

**Review by shhossain — APPROVED** [review]

**Pull request merged**